react/src/components/CarouselCards.jsx

```diff
@@ -1,17 +1,17 @@
-import React from "react";
-import Carousel from "react-material-ui-carousel";
-import slider from "../api/slider.json";
-import Item from "./Item";
+// import React from "react";
+// import Carousel from "react-material-ui-carousel";
+// import slider from "../api/slider.json";
+// import Item from "./Item";
 
-export default function CarouselSlide({ onInput }) {
-  return (
-    <Carousel>
-      {slider.map((item) => (
-        <Item key={item.id} item={item} onInput={onInput} />
-      ))}
-    </Carousel>
-  );
-}
+// export default function CarouselSlide({ onInput }) {
+//   return (
+//     <Carousel>
```

Comment:
@FennyWilriani You commented CarouselCard element so it doesn't work.
Please check and make sure that all parts are working before you pull your changes.

Reply:
Yes, I had to comment on the updated code as I have an error displaying the result on react server, will fix it in the moment.